source/material_model/rheology/elasticity.cc

```diff
@@ -146,8 +147,27 @@ namespace aspect
         if (this->convert_output_to_years())
           fixed_elastic_time_step *= year_in_seconds;
 
+        // When using the visco_plastic material model,
```

Comment:
I think this issue also applies to the viscoelastic material as well?

Reply:
The viscoelastic material model will also have to be updated due to changes in elasticity.cc, but I haven't done that yet. First want to be convinced vp works!